### PR TITLE
correct the description and make the context consistent

### PR DESCRIPTION
**Pull request by yin1999**
### Description

correct the description and make the context consistent

### CODE REVIEW

File: files/en-us/web/api/subtlecrypto/wrapkey/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Wrapping a key helps protect it in untrusted environments, such as inside an oth
 As with {{DOMxRef("SubtleCrypto.exportKey()")}}, you specify an [export format](/en-US/docs/Web/API/SubtleCrypto/importKey#supported_formats) for the key.
 To export a key, it must have {{DOMxRef("CryptoKey.extractable")}} set to `true`.
 
-But because `wrapKey()` also encrypts the key to be imported, you also need to pass in the key that must be used to encrypt it.
+But because `wrapKey()` also encrypts the key to be exported, you also need to pass in the key that must be used to encrypt it.
```
Comment (yin1999) [inline files/en-us/web/api/subtlecrypto/wrapkey/index.md:18]: `wrapKey()` is used to export and encrypt the key.

File: files/en-us/web/api/subtlecrypto/wrapkey/index.md [hunk 2]
```diff
@@ -58,10 +58,9 @@ wrapKey(format, key, wrappingKey, wrapAlgo)
 
 ### Return value
 
-- `result` is a [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
-  that fulfills with an
-  [`ArrayBuffer`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer)
-  containing the encrypted exported key.
+A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that fulfills with
+an [`ArrayBuffer`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer)
+containing the encrypted exported key.
```
Comment (yin1999) [inline files/en-us/web/api/subtlecrypto/wrapkey/index.md:63]: Make this just like the description in other crypto methods

File: files/en-us/web/api/subtlecrypto/wrapkey/index.md [hunk 3]
```diff
@@ -96,7 +95,7 @@ AES-KW is specified in [RFC 3394](https://datatracker.ietf.org/doc/html/rfc3394)
 ### Raw wrap
 
 This example wraps an AES key.
-It uses "raw" as the export format and AES-KW, with a password-derived key, to encrypt it. [See the complete code on GitHub](https://github.com/mdn/dom-examples/blob/main/web-crypto/wrap-key/raw.js).
+It uses "raw" as the export format and AES-KW, with a password-derived key, to encrypt it. [See the complete code on GitHub.](https://github.com/mdn/dom-examples/blob/main/web-crypto/wrap-key/raw.js)
```
Comment (yin1999) [inline files/en-us/web/api/subtlecrypto/wrapkey/index.md:98]: following the below example descriptions

File: files/en-us/web/api/subtlecrypto/wrapkey/index.md [hunk 4]
```diff
@@ -328,9 +327,9 @@ window.crypto.subtle
   .then((wrappedKey) => console.log(wrappedKey));
 ```
 
-### JSON Web Key import
+### JSON Web Key wrap
 
-This code wraps an ECDSA private signing key. It uses "jwk" as the export format and AES-GCM, with a password-derived key, to encrypt it.
+This example wraps an ECDSA private signing key. It uses "jwk" as the export format and AES-GCM, with a password-derived key, to encrypt it.
```
Comment (yin1999) [inline files/en-us/web/api/subtlecrypto/wrapkey/index.md:332]: follow the above examples

**Comment by yin1999** [comment]
Is there any one who could review this? This blocks the [localization of this page](https://github.com/mdn/translated-content/pull/13364)

**Review by Josh-Cena — APPROVED** [review]
LGTM. Thanks :)

**Pull request merged**